src/acquisition/covid_hosp/state_daily/database.py

```diff
@@ -0,0 +1,83 @@
+# first party
+from delphi.epidata.acquisition.covid_hosp.common.database import Database as BaseDatabase
+from delphi.epidata.acquisition.covid_hosp.common.utils import Utils
+
+
+class Database(BaseDatabase):
+
+  TABLE_NAME = 'covid_hosp_state_daily'
```

Comment:
tricky; we should discuss whether to open a new table for this dataset or figure out a scheme for losslessly storing it in the timeseries table

Reply:
yes, good to think about. in the interest of capturing this dataset "as is", i lean towards a new table, especially since (i might be mistaken) it looks like there are some fields/columns in this dataset that don't exist in the "state timeseries" dataset.

Reply:
(i suspect any new indicators should probably use a combination of weekly "state timeseries" and daily "state daily" to make a combo signal that's based on the most up-to-date union of fields from the two datasets)